internal/services/authorization/role_assignment_resource_test.go

```diff
@@ -627,3 +641,19 @@ resource "azurerm_role_assignment" "test" {
 }
 `, data.RandomInteger, data.Locations.Primary)
 }
+
+func (RoleAssignmentResource) capacityProviderScoped() string {
+	return `
+provider "azurerm" {
+  features {}
+}
+
+data "azurerm_client_config" "test" {}
+
+resource "azurerm_role_assignment" "test" {
+  scope                = "/providers/Micrososoft.Capacity"
```

Comment:
```suggestion
  scope                = "/providers/Microsoft.Capacity"
```

Reply:
@stephybun, you're right, I'm going to fix that. Nevertheless the provider needs to be registered on the global scope (/providers/) not on subscription scope (/subscriptions/.../providers/)